server/src/main/java/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -1272,15 +1273,15 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
 
         String value = _configDao.getValue(Config.BackupSnapshotWait.toString());
 
-        Type.HOURLY.setMax(SnapshotHourlyMax.value());
-        Type.DAILY.setMax(SnapshotDailyMax.value());
-        Type.WEEKLY.setMax(SnapshotWeeklyMax.value());
-        Type.MONTHLY.setMax(SnapshotMonthlyMax.value());
+        Type.HOURLY.setMax(snapshotHourlyMax.value());
+        Type.DAILY.setMax(snapshotDailyMax.value());
+        Type.WEEKLY.setMax(snapshotWeeklyMax.value());
+        Type.MONTHLY.setMax(snapshotMonthlyMax.value());
         _totalRetries = NumbersUtil.parseInt(_configDao.getValue("total.retries"), 4);
         _pauseInterval = 2 * NumbersUtil.parseInt(_configDao.getValue("ping.interval"), 60);
 
-        snapshotBackupRetries = BackupRetryAttempts.value();
-        snapshotBackupRetryInterval = BackupRetryInterval.value();
+        snapshotBackupRetries = backupRetryAttempts.value();
```

Comment:
'retries' would be 'attempts' minus 1, would it?

Reply:
@DaanHoogland 
1. #3035 thorough description added

Reply:
@DaanHoogland this is no my code, don't dive into semantics. What I did is tjust renamed variable to comply normal camel case style with refactoring utility.

If I didn't get what you mean, please try to explain me one more time)

Reply:
ok, not your doing, @bwsw. rather ugly misuse of English though: number of attempts == initial attempt plus number of retries.